### PR TITLE
fix: update button style to support ms-high-contrast

### PR DESCRIPTION
**Pull request by nalogiudice**
Closes [#1715](https://github.com/microsoft/fast-dna/issues/1715)

# Description

Updates button style to support ms-high-contrast.

Screen Shot 2019-06-03 at 12 54 55 PM

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-components-styles-msft/src/button/index.ts [hunk 1]
```diff
@@ -235,6 +248,11 @@ const styles: ComponentStyles<ButtonClassNameContract, DesignSystem> = {
     },
     button__disabled: {
         ...applyDisabledState(),
+        "& $button_beforeContent, & $button_afterContent": {
+            "@media (-ms-high-contrast:active)": {
+                fill: "white", // ButtonHighlight cannot be passed to -ms-high-contrast but white is safe in this context
```
Comment (chrisdholt) [inline packages/fast-components-styles-msft/src/button/index.ts]: Can you elaborate on this? I see `ButtonHighlight` being passed above to a -ms-high-contrast query. My concern with "white" is that there are multiple high contrast modes supported by Windows and white may not meet contrast in all of them. Can we confirm these worked in all of the modes?

Comment (nalogiudice) [reply packages/fast-components-styles-msft/src/button/index.ts]: The compound JSS statement was causing the issue. Separating `& $button_beforeContent` and `& $button_afterContent` allowed the usage of `ButtonHighlight`.

File: packages/fast-components-styles-msft/src/button/index.ts [hunk 2]
```diff
@@ -235,6 +248,12 @@ const styles: ComponentStyles<ButtonClassNameContract, DesignSystem> = {
     },
     button__disabled: {
         ...applyDisabledState(),
+        "& $button_beforeContent": {
```
Comment (nicholasrice) [inline packages/fast-components-styles-msft/src/button/index.ts]: can you chain these? `& $button_beforeContent, & $button_afterContent`

Comment (nalogiudice) [reply packages/fast-components-styles-msft/src/button/index.ts]: See my above comment to @chrisdholt - not sure why but chaining them doesn't allow `ButtonHighlight` to be passed.

**Pull request merged**